charts/enterprise-kyverno-operator/values.yaml

```diff
@@ -1,4 +1,4 @@
-# Default values for enterprise-kyverno.
+# Default values for nirmata-kyverno.
```

Comment:
Is the repo listed in this file `repository: ghcr.io/nirmata/enterprise-kyverno-operator` correct?

Reply:
Yes. The problem is, there is another image (old operator, the first version, even before we started calling it a monitor), whose name is nirmata-kyverno-operator. And owned by the nirmata/kyverno-operator repo.